misc/utils/ReadSizes.java

```diff
@@ -40,9 +41,48 @@ private static void scanExistingFiles(File dir) {
     }
   }
 
+  private static void scanTilesIndex(String tilesUrl) {
+    try {
+      URL url = new URL(tilesUrl);
+      HttpURLConnection con = (HttpURLConnection) url.openConnection();
+      con.setRequestMethod("GET");
+      BufferedReader in = new BufferedReader(
+        new InputStreamReader(con.getInputStream()));
+      String inputLine;
+      while ((inputLine = in.readLine()) != null) {
+        parseAndUpdateTileSize(inputLine);
+      }
+      in.close();
+    } catch (MalformedURLException e) {
+      System.out.println("Invalid URL");
+    } catch (ProtocolException e) {
+      System.out.println("Unable to download segment index");
+    } catch (IOException e) {
+      System.out.println("Unable to download segment index");
+    }
+  }
+
+  private static void parseAndUpdateTileSize(String indexLine) {
```

Comment:
I think it is good practice to document code that does parsing using a sample of what should be parsed, in this case:

`<a href="E0_N35.rd5">E0_N35.rd5</a>                                         17-Oct-2021 01:03            1290596`

Also mentioning that (typically) https://brouter.de/brouter/segments4/ is used would be good.

Reply:
Thanks, that's good to have!